python_modules/dagster/dagster_tests/definitions_tests/test_time_window_partitions.py

```diff
@@ -1228,6 +1228,19 @@ def my_partitioned_config_2(_start, _end):
         == partitions_def.get_partition_keys(current_time=current_time)[50:53]
     )
 
+    partitions_def = TimeWindowPartitionsDefinition(
```

Comment:
My main concern with this sort of scheme is how it handles DST weirdnesses and leap years etc. It'd be great to add some tests which span larger time ranges to ensure that we have coverage for those sorts of things

Reply:
took out the end time so now it includes both! (not really an issue for these types of cronstrings since you can always just advance the time by that many minutes, but good to be certain for sure)